tests/test_canvas.py

```diff
@@ -75,7 +76,7 @@ def test(self):
         self.assertEquals(future.count_finished_activities, 2)
         self.assertEquals(future._result, ["test1", None, 3])
         with self.assertRaises(exceptions.ExecutionBlocked):
-            future.result
+            dummy = future.result
```

Comment:
`this_is_going_to_raise = future.result`?
:trollface: This lowers PyCharm's warning level